apps/autoscheduler/AutoSchedule.cpp

```diff
@@ -2650,9 +2697,21 @@ struct State {
     }
 
     void dump() const {
-        aslog(0) << "State with cost " << cost << ":\n";
+        if (aslog::aslog_level() == 0) {
```

Comment:
Not sure I understand this part. It just prints which Funcs are compute_root?

Reply:
Yeah, pretty much. I'm not sure that there is a good, terse way to describe each pass; this was my arbitrary first stab at it. Maybe we'd be better off just skipping that entirely and saying `Pass N of M, Cost Foo`?

Reply:
Yeah, I think that's sufficient. The logging here is a sketch of the loop nest. There isn't really a useful condensed version of that.